runners/flink/flink_runner.gradle

```diff
@@ -239,8 +239,8 @@ dependencies {
     exclude group: "org.apache.logging.log4j", module: "log4j-core"
   }
   implementation "org.apache.flink:flink-connector-kafka:1.17-lyft202402051707167466"
-  implementation "com.lyft:streamingplatform-kinesis:2.4.1-20240214.184504-6"
-  implementation ("com.lyft:streamingplatform-kafka:2.4.1-20240214.184504-6") {
+  implementation "com.lyft:streamingplatform-kinesis:2.4.1-20240416.211155-8"
+  implementation ("com.lyft:streamingplatform-kafka:2.4.1-20240416.211155-8") {
```

Comment:
Why is kafka updated?

Reply:
Thought it would be cleaner to keep them all on the same version, even if the relevant change is only for kinesis